SU-HSDO/suhumsci
# feat(STN-344): Traditional Theme Spacing

# READY FOR REVIEW

## Summary
Adjusts and adds a theme-wide system for spacing variables.

For the traditional theme, we amended the spacing for things like:

- gutters and margin
- spacing between components
- spacing between fields on detail pages
- fields in UI patterns

I didn't address headings or wells in this card - it would be better if Merani could do this work in the browser and to polish there.

## Steps to Test
1. Ensure `npm run test` works in humsci_basic
2. Take a look at a variety of pages, ensure that nothing, such as grids, or cards, or WYSIWYG areas look broken with regards to spacing.
3. See the tests I wrote for the new spacing function and if you agree

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
